src/common/processing_linux.c

```diff
@@ -20,40 +20,52 @@ static inline void waitpid_wrapper(const pid_t* pid)
         waitpid(*pid, NULL, 0);
 }
 
+static inline int ffPipe2(int *fds, int flags) {
+    #ifdef __APPLE__
+        if(pipe(fds) == -1)
+            return -1;
+        fcntl(fds[0], F_SETFL, fcntl(fds[0], F_GETFL) | flags);
+        fcntl(fds[1], F_SETFL, fcntl(fds[1], F_GETFL) | flags);
+        return 0;
+    #else
+        return pipe2(fds, flags);
+    #endif
+}
+
 const char* ffProcessAppendOutput(FFstrbuf* buffer, char* const argv[], bool useStdErr)
 {
     int pipes[2];
+    const int timeout = instance.config.general.processingTimeout;
+    const int pipeFlags = O_CLOEXEC;
```

Comment:
Don't make it a separate variable. It's used only once and `const` in `C` doesn't really make a variable a constant.